include/mcptam/KeyFrame.h

```diff
@@ -390,7 +397,13 @@ class MultiKeyFrame
 
   void RemoveImages();
 
+  //partially copy MultiKeyFrame
+  
+  MultiKeyFrame* CopyMultiKeyFramePartial(); //returns a pointer to the partially copied MKF
+  
   // Variables
+  bool isBufferMKF;  ///am I a buffer MKF?  If so, we'll use the flag to disable things such as updating measurements to the global points list when buffering keyframe information
```

Comment:
Pretty obvious flag, does not need a comment. Comments on what we'll do with it should go where we use this flag.